proto/elys/tier/userdata.proto

```diff
@@ -23,10 +23,11 @@ message Pool {
 }
 
 enum MembershipTierType {
-  BRONZE = 0;
-  SILVER = 1;
-  GOLD = 2;
-  PLATINUM = 3;
+  BASIC = 0;
+  BRONZE = 1;
```

Comment:
Do we need to migrate previous data because of this?

Reply:
no, this data is only used in query